requirements.dev.txt

```diff
@@ -0,0 +1 @@
+pyOpenSSL==16.2.0
```

Comment:
I wouldn't require a specific version here. It's dev requirements, having the latest one is almost always a safe  choice.